Decred Wallet/Assets.xcassets/nav_menu/ic_overview.imageset/Contents.json

```diff
@@ -2,15 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
+      "filename" : "overview.png",
       "scale" : "1x"
     },
     {
       "idiom" : "universal",
-      "filename" : "overviewicon3.png",
```

Comment:
Is `overviewicon3.png` deleted? It should be, unless it's used elsewhere.

Reply:
renamed and resolved in 37e2453